_data/guides.yml

```diff
@@ -58,6 +58,8 @@ toc:
     path: /docs/user-guide/config-best-practices/
   - title: Using kubectl to Manage Resources
     path: /docs/user-guide/working-with-resources/
+  - title: Garbage collection
```

Comment:
We also have Garbage collection for container images.  Is a way to disambiguate?